### DISCUSSION TITLE
Using standalone UserOperator for connecting to MSK cluster using SASL/SCRAM

### DISCUSSION THREAD

**Discussion by guptatapan** [strimzi/strimzi-kafka-operator#6998] [Q&A] [answered]
Hello,

We wanted to deploy User Operator to connect to MSK which is configured with SASL/SCRAM authentication.
I was going through the docs : https://strimzi.io/docs/operators/latest/full/deploying.html#deploying-the-user-operator-standalone-str but could not find any setting, on how to provide SASL properties to User Operator.

These properties are there in TopicOperator though, so is it only supported in Topic and not in User Operator?

**Comment by scholzj** [comment, marked as answer]
The User and Topic operators are primarily developed to be used with Strimzi based Kafka clusters where they use TLS client authentication. So it might not have the option to use SASL authentication and you would need to open a PR with it if you want it.

**Comment by scholzj** [comment]
FYI: While trying to address something else, I opened https://github.com/strimzi/strimzi-kafka-operator/pull/7019 which if accepted / approved might make this possible in Strimzi 0.30.

**Reply by avestuk** [reply]
It should be noted that MSK requires that users are configured via AWS Secrets Manager for SASL/SCRAM authentication.
